README.md

```diff
@@ -17,4 +17,169 @@
 - Booleans, Comparison Operators, Flow Control 
 - Arrays
 
+
+## Strolling through the zoo
+Imagine you're walking in the zoo. Now, take a look around you. Aren't those lions, parrots and elephants awesome? Wouldn't it be cool if, in our code, we could catalogue the animals we see? To do that, we need to take a moment to talk about data structures. Specifically, how we would store that data in an **object**.
+
+Let's use the lion as an example. Try and describe it in your head. Think of any relevant information we might care about. We can come up with several properties for the lion: the weight, its age, its name, and so on. Wait a minute; let's back up to that previous sentence real quick — there's an important word there. *Properties.* An object is basically a bunch of properties grouped together, kind of like a filing cabinet.
+
+A property has a name (a *key*) that is connected to a value. For example, our lion object would have a `weight` key, which has a value of `190` (thanks, Google!). Of course, we're using kilograms here to express the lion's weight, since the metric system is *clearly* the superior system — European bias notwithstanding.
+
+
+## A data structure fit for a king
+The most common way to create an object is to use the *object literal* notation, as shown here:
+
+```js
+const animal = {};
+```
+
+Woo, we've created our first object! However, right now, it's pretty boring. It has no properties whatsoever. That's pretty useless. We can add properties to an object after it's been created, which we'll talk about later. But for now, let's create our animal again — this time adding a property:
+
+```js
+const animal = {
+  species: 'lion'
+};
+```
+
+Now we're talking. Take a moment to study the syntax a bit, and think about what we just did. We added a property to our object when we created it. If you remember, a property has both a *key* and a *value*. In this case, our object has a property with the key `species` and the string `'lion'` as its value.
+
+You might be way ahead of me already, but let's create a lion out of *thin air*, with some more relevant properties. Ready? Go!
+
+```js
+const simba = {
+  species: 'lion',
+  name: 'Simba',
+  weight: 190,
+  age: 8
+};
+```
+
+A value can be of any type. This means that we can store strings, numbers, booleans, arrays and functions in our object. We can even add other objects as values to our object! An image of [Leonardo Di Caprio looking at a gravity defying cityscape](https://www.wired.com/images_blogs/underwire/2010/07/inception_paris_660.jpg) springs to mind, but I digress. 
+
+```js
+const simba = {
+  species: 'lion',
+  name: 'Simba',
+  weight: 190,
+  age: 8,
+  diet: ['antelope', 'tortoise', 'zebra'],
```

Comment:
We might want to hold off on throwing too much at students at once. Assigning strings, numbers, and booleans to objects to start helps to introduce the main points while avoiding some expensive conceptual overhead (a future lesson could cover complex data structures in objects).